azure-pipelines.yml

```diff
@@ -33,6 +33,11 @@ jobs:
           ARCHITECTURE: x64
           CONFIGURATION: Release
           WINSTORE: -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION="10.0.17763.0"
+        ARM-UWP:
```

Comment:
it will not being compiled for arm becaue ns-eel2 has no asm for arm

Reply:
Aha, thanks :smile:.
Thought by the use of `./lib/vis_milkdrop/ns-eel2/asm-nseel-arm.asm` that there something about is in.

Then better to remove the ARM build complete?

Reply:
iirc there is no asm compiler for arm-uwp platform, so yes remove arm-uwp build completely